packages/remote-config/lib/index.js

```diff
@@ -93,21 +135,47 @@ class FirebaseConfigModule extends FirebaseModule {
     }
 
     if (!hasOwnProperty(this._values, key)) {
-      return {
-        value: undefined,
+      return new Value({
+        value: '',
         source: 'static',
-      };
+      });
     }
 
-    return this._values[key];
+    return new Value({ value: `${this._values[key].value}`, source: this._values[key].source });
   }
 
   getAll() {
-    return Object.assign({}, this._values);
+    const values = {};
+
+    Object.keys(this._values).forEach(key => (values[key] = this.getValue(key)));
+
+    return values;
+  }
+
+  get defaultConfig() {
+    console.warn(
+      'firebase.remoteConfig().defaultConfig is not supported. Default values are merged with config values',
+    );
+  }
+
+  set defaultConfig(defaults) {
+    console.warn(
+      'firebase.remoteConfig().defaultConfig is not supported. Please use firebase.remoteConfig().setDefaults({ [key] : value }) to set default values',
+    );
+  }
+
+  get settings() {
+    return this._settings;
+  }
+
+  set settings(settings) {
+    console.warn(
+      "firebase.remoteConfig().settings = { [key]: string }; is not supported. Please use 'firebase.remoteConfig().settings = { ...[key]: string, }' instead'",
```

Comment:
Doesn't make sense, is saying 'use the same api instead' - guessing this should be saying use setConfigSettings instead